### DISCUSSION TITLE
Do I need to use --reinstall to get torch 2.0.1?

### DISCUSSION THREAD

**Discussion by usigg3** [vladmandic/sdnext#954] [Q&A]
Or how do I do it?

**Comment by Averytingwong** [comment]
--reinstall just installed 2.0.1 for me

**Reply by usigg3** [reply]
Tried --reinstall but got some errors on torch install... Maybe I need to reboot or something...

**Comment by vladmandic** [comment]
typically, yes, `--reinstall` should do the trick. but "some errors" is not something i can diagnose.

**Comment by derspanier1** [comment]
i am considering a fresh reinstall via flag. could someone briefly give me short bullets what the most painless way would be ? backup config.json and ui-config.json, styles.csv ? thanks for help.

**Reply by vladmandic** [reply]
reinstall will not delete those 3, it will reinstall requirements only - its not the same as fresh install (which would also check out newer versions of upstream repos where possible).
also, new versions do have new features that you may miss out on if you just carry over old config (for example, there is a new default live preview mode, but reinstall will keep whatever was previously set).
but definitely keep your `styles.csv`!

**Reply by derspanier1** [reply]
thanks for info ! ill just create a separate folder and make a fresh install.
